featuretools/tests/dfs_tests/test_deep_feature_synthesis.py

```diff
@@ -694,6 +694,33 @@ def test_initialized_agg_prim(es):
     assert (feature_with_name(features, "N_MOST_COMMON(log.product_id)"))
 
 
+def test_return_variable_types(es):
+    dfs_obj = DeepFeatureSynthesis(target_entity_id="sessions",
+                                   entityset=es,
+                                   agg_primitives=[Count, NMostCommon],
+                                   trans_primitives=[Absolute, Hour, IsIn])
+
+    discrete = ft.variable_types.Discrete
+    numeric = ft.variable_types.Numeric
+
+    f1 = dfs_obj.build_features(return_variable_types=None)
```

Comment:
we should add a test where you go outside of the default type. for example, if you pass in `Datetime` or `DatetimeTimeIndex` it should return some additional features since sessions has datetime columns.